libhb/handbrake/common.h

```diff
@@ -563,6 +563,7 @@ struct hb_job_s
 #define HB_VCODEC_FFMPEG_VP9   0x0000080
 #define HB_VCODEC_FFMPEG_VCE_H264 0x00040000
 #define HB_VCODEC_FFMPEG_VCE_H265 0x00080000
+#define HB_VCODEC_FFMPEG_VCE_H265_10BIT 0x00080001
```

Comment:
HB_VCODEC_ is kind of a bit field, and each codec should be a single bit. But I'll have to reassign them anyway for the VP9 10 bit pull request, so let's wait a bit to fix this.